RadixWallet/Features/AppFeature/SceneDelegate.swift

```diff
@@ -58,6 +79,28 @@ public final class SceneDelegate: NSObject, UIWindowSceneDelegate, ObservableObj
 		self.overlayWindow = overlayWindow
 	}
 
+	// MARK: Biometrics
+
+	private var biometricsSplashWindow: UIWindow?
+
+	private func showBiometricsSplashWindow(in scene: UIWindowScene) {
+		let splashView = Splash.View(
```

Comment:
Why not reuse the `privacyProtectionWindow`?

Reply:
I see the `privacyProtectionWindow` as a completely separate layer. It has a different windowLevel and context, and its purpose is to cover all other windows when the app is in App Switcher mode, including the biometricsSplashWindow if it's displayed at that time.